config/config.go

```diff
@@ -17,4 +17,5 @@ package config
 const (
 	OperatorName      string = "gcp-project-operator"
 	OperatorNamespace string = "gcp-project-operator"
+	EnableOLMSkipRange string = "true"
```

Comment:
This should fail linting, but either way can you align these? You can also just omit `string` and let the type system infer that. 

```golang
package config

const (
	OperatorName       = "gcp-project-operator"
	OperatorNamespace  = "gcp-project-operator"
	EnableOLMSkipRange = "true"
)
```